Index: src/qmmp/qmmp.cpp
===================================================================
--- src/qmmp/qmmp.cpp	(revision 13279)
+++ src/qmmp/qmmp.cpp	(revision 13280)
@@ -74,7 +74,7 @@
 QString Qmmp::cacheDir()
 {
 #ifdef Q_OS_WIN
-    return *::configDir();
+    return Qmmp::configDir();
 #else
     if(::configDir()->isEmpty())
         return QStringLiteral("%1/%2").arg(QStandardPaths::writableLocation(QStandardPaths::GenericCacheLocation),
Index: .
===================================================================
--- .	(revision 13279)
+++ .	(revision 13280)

Property changes on: .
___________________________________________________________________
Modified: svn:mergeinfo
## -0,0 +0,1 ##
   Merged /trunk/qmmp:r13279
